integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java

```diff
@@ -191,9 +191,10 @@ private static boolean assertIfLogExists(LogViewerClient logViewerClient, String
 
     /**
      * Check whether a log found with expected string of given priority
+     *
      * @param logViewerClient LogViewerClient object
-     * @param priority priority level
-     * @param expected expected string
+     * @param priority        priority level
```

Comment:
We don't format the java doc comment with the tab

Reply:
By auto correct all params are alined to same level. Not formatted by a tab.